bitchat/Models/CommandsInfo.swift

```diff
@@ -0,0 +1,60 @@
+//
+// CommandsInfo.swift
+// bitchat
+//
+// This is free and unencumbered software released into the public domain.
+// For more information, see <https://unlicense.org>
+//
+
+import Foundation
+
+// MARK: - CommandInfo Enum
+
+enum CommandInfo: String, CaseIterable, Identifiable {
+    case block = "/block"
+    case clear = "/clear"
+    case hug = "/hug"
+    case message = "/m, /msg"
+    case slap = "/slap"
+    case unblock = "/unblock"
+    case who = "/w"
+    case favorite = "/fav"
+    case unfavorite = "/unfav"
```

Comment:
I'd probably name the cases as the commands without having to declare the actual string values (as to avoid future syncing issues) - and then just add a `/` prefix when comparing